src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java

```diff
@@ -900,4 +890,23 @@ public static void enterKeyValuePairsForProperty(String pluginProperty, String j
       index++;
     }
   }
+  /**
+   * Enter Credentials/Authorization values in the Plugin Property (input)
+   *
+   * @param pluginProperty @data-cy attribute value of Plugin Property.
+   *                       If pluginProperty is present in {@link ConstantsUtil#DEFAULT_DATACY_ATTRIBUTES_FILE}
+   *                       then its data-cy is fetched from it
+   *                       else pluginProperty is used as it is.
+   * @param envVariableKey          If the value is present in the Environment variables then its value is fetched
+   */
+  public static void enterValueInInputPropertyFromEnv(String pluginProperty, String envVariableKey) {
+    WebElement pluginPropertyInput = PluginPropertyUtils.getInputPluginPropertyElement(pluginProperty);
+
+    if (System.getenv(PluginPropertyUtils.pluginProp(envVariableKey)) != null) {
```

Comment:
Can we store the `System.getenv(PluginPropertyUtils.pluginProp(envVariableKey)` in local variable and reuse it?